src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3822,7 +3822,11 @@ private static bool CanReuseParameter(CSharp.Syntax.ParameterSyntax parameter)
             {
                 equals = this.EatToken(SyntaxKind.EqualsToken);
             }
-            exclamation = CheckFeatureAvailability(exclamation, MessageID.IDS_ParameterNullChecking);
+            if (!(exclamation is null))
```

Comment:
This doesn't appear to do anything

Reply:
Consider just doing `_ = exclamation is null ? null : CheckFeatureAvailability`, or prefer just making this an `if` rather than an assignment.

---
In reply to: [308933293](https://github.com/dotnet/roslyn/pull/37585#discussion_r308933293) [](ancestors = 308933293)